stdlib/math.jou

```diff
@@ -1,40 +1,107 @@
+# A full turn (360 degrees) is 2pi (about 6.28) radians.
+declare cos(x: double) -> double
+declare sin(x: double) -> double
+declare tan(x: double) -> double
+# The 'a' prefixed functions are inverse functions.
+# For example, asin is also known as arcsin and sin^-1.
 declare acos(x: double) -> double
 declare asin(x: double) -> double
 declare atan(x: double) -> double
+# Returns the angle of a point(x, y). Note the reversed order of the arguments.
 declare atan2(y: double, x: double) -> double
-declare cos(x: double) -> double
-declare sin(x: double) -> double
-declare tan(x: double) -> double
 
-declare fmod(x: double, y: double) -> double
-declare sqrt(x: double) -> double
-declare cbrt(x: double) -> double
+# Floating-point manipulation functions
+# Returns a value with the magnitude of x and the sign of y
+# Example:
+#   printf("%f", copysign(4, -1))
+# Output:
+#   -4.0
+declare copysign(x: double, y: double) -> double
+# Returns a quiet NaN (Not-A-Number) value of type .
+# The NaN values are used to identify undefined or non-representable values for floating-point elements, such as the square root of negative numbers or the result of 0/0.
+# The argument can be used by library implementations to distinguish different NaN values in a implementation-specific manner.
+declare nan(tagp: byte*) -> double
+# Returns the next representable value after x in the direction of y.
+declare nextafter(x: double, y: double) -> double
 
-declare log(x: double) -> double
-declare log10(x: double) -> double
-declare log2(x: double) -> double
+# Hyperbolic functions
+# Returns the hyperbolic cosine of x.
+declare cosh(x: double) -> double
+# Returns the hyperbolic sine of x.
+declare sinh(x: double) -> double
+# Returns the hyperbolic tangent of x.
+declare tanh(x: double) -> double
+# Returns the nonnegative area hyperbolic cosine of x.
+declare acosh(x: double) -> double
+# Returns the area hyperbolic sine of x.
+declare asinh(x: double) -> double
+# Returns the area hyperbolic tangent of x.
+declare atanh(x: double) -> double
 
+# Exponential and logarithmic functions
+# Returns the base-e exponential function of x
 declare exp(x: double) -> double
 declare exp2(x: double) -> double
-declare pow(x: double, y: double) -> double
+# Returns the result of multiplying x (the significand) by 2 raised to the power of exp (the exponent).
 declare ldexp(x: double, exp: int) -> double
-
+# Returns the natural logarithm of x.
+declare log(x: double) -> double
+declare log10(x: double) -> double
+declare log2(x: double) -> double
+# Returns e raised to the power x minus one
 declare fmax(x: double, y: double) -> double
 declare fmin(x: double, y: double) -> double
+# Returns the positive difference between x and y.
+declare fdim(x: double, y: double) -> double
+# Returns e raised to the power x minus one
+declare expm1(x: double) -> double
+# Returns the natural logarithm of one plus x.
+declare log1p(x: double) -> double
```

Comment:
Once again, I don't see how these functions are useful, because it's easy to achieve what they do with other functions.

```suggestion
```